src/ceph-volume/ceph_volume/tests/util/test_device.py

```diff
@@ -150,6 +150,42 @@ def test_is_member_blkid(self, device_info, label):
 
         assert disk.is_member is True
 
+    def test_no_removable_device(self, device_info):
+        data = {"/dev/sdb": {"removable": 1}}
+        device_info(devices=data)
+        disk = device.Device("/dev/sdb")
+        assert not disk.is_valid
+
+    def test_no_removable_device2(self, device_info):
```

Comment:
`test_no_removable_device2` doesn't really convey anything, try to use meaningful naming with these tests. For example this could be: `test_non_removable_device_is_valid`, although you may find other better options

Reply:
done.

Reply:
done.